lifecycle_msgs/msg/Transition.msg

```diff
@@ -45,5 +45,14 @@ uint8 TRANSITION_ON_ERROR_SUCCESS = 60
 uint8 TRANSITION_ON_ERROR_FAILURE = 61
 uint8 TRANSITION_ON_ERROR_ERROR = 62
 
+# Three return values which ought to be set
+# as a return value for each callback.
+# Depending on which return value,
+# the transition will be executed correctly
+# or failback/error callbacks will be triggered.
+uint8 TRANSITION_CALLBACK_SUCCESS = 99
+uint8 TRANSITION_CALLBACK_FAILURE = 98
+uint8 TRANSITION_CALLBACK_ERROR = 97
```

Comment:
Any reason for using decreasing error codes rather than increasing as for the other codes in this message definition ? 
I would have expected (to stay consistent):
```
uint8 TRANSITION_CALLBACK_SUCCESS = 97
uint8 TRANSITION_CALLBACK_FAILURE = 98
uint8 TRANSITION_CALLBACK_ERROR = 99
```